src/components/PlanDropdown.jsx

```diff
@@ -38,4 +41,14 @@ const PlanDropdown = ({ plans = ["Second Major", "without honours"], selected =
   );
 };
 
-export default PlanDropdown;
+const mapStateToProps = (state) => ({
```

Comment:
Does this mean we'll know what plan the user is using?

Reply:
Yea it'll be passed into the `PlanDropdown` component state as the `selected` plan.